Library/Homebrew/test/dev-cmd/tap-new_spec.rb

```diff
@@ -7,6 +7,13 @@
   it_behaves_like "parseable arguments"
 
   it "initializes a new tap with a README file and GitHub Actions CI", :integration_test do
+    # To ensure that Utils::Git.setup_gpg! doesn't raise an error
+    setup_test_formula "gnupg", <<~RUBY
+      desc "GNU Pretty Good Privacy (PGP) package"
+      homepage "https://gnupg.org"
+      url "https://gnupg.org/ftp/gcrypt/gnupg/gnupg-2.2.27.tar.bz2"
+    RUBY
```

Comment:
```suggestion
    setup_test_formula "gnupg"
```

If this works I think it's preferable.

Reply:
Done, I had to modify `integration_test.rb` to get it to work, similar to `patchelf`.